libutils/definitions.h

```diff
@@ -44,4 +44,11 @@
 #define CF_BUFSIZE     4096
 #define CF_EXPANDSIZE (2 * CF_BUFSIZE)
 
+
+/*****************************************************************************
+ *                          File permissions                                 *
+ *****************************************************************************/
+#define CF_MODE_DEFAULT  S_IWUSR | S_IRUSR
+#define CF_MODE_SHARED   CF_MODE_DEFAULT | S_IRGRP | S_IROTH
```

Comment:
Since mode means something different for `open()` and `fopen()` I would prefer something more clear/explicit:

```C
// 600 - Read/Write for owner:
#define CF_CREATE_PERMS_DEFAULT S_IWUSR | S_IRUSR
// 644 - World readable:
#define CF_CREATE_PERMS_WORLD_READABLE CF_CREATE_PERMS_DEFAULT | S_IRGRP | S_IROTH
```

Reply:
In the branch in my fork I changed them to `CF_PERMS_DEFAULT` and `CF_PERMS_SHARED`. I think it might be better to use `S_IWUSR` and similar instead, though.